Relation-SPA/src/app/relation-table/relation-table.component.ts

```diff
@@ -1,22 +1,40 @@
 import { Component, OnInit } from '@angular/core';
 import { RelationService } from '../_services/relation.service';
+import { Relation } from '../_models/relation';
+import { Category } from '../_models/category';
 
 @Component({
   selector: 'app-relation-table',
   templateUrl: './relation-table.component.html',
   styleUrls: ['./relation-table.component.scss']
 })
 export class RelationTableComponent implements OnInit {
-  ralation: any = [];
+  relations: Relation[];
+  categoryList: any = [];
+  categories: Category[];
   constructor(private _relationService: RelationService) { }
 
   ngOnInit() {
-
+    this._relationService.GetCategorys().subscribe(_ => {
+      this.categoryList = _;
+      this.categories = _;
+      console.log(this.categories);
+      console.log(this.categoryList);
```

Comment:
console.log - should be removed, bad practice to push them

Reply:
console.log was removed